---
geometry_msgs/PoseArray waypoints
int32 waypoints_count
bool success
string message